### PR TITLE
Add head code to prevent Declare JS Vars bug

### LINKED ISSUE #7806: Bug in Solution
I think according to the problem statement, the problem should accept the solution a=7 and then b=a statements. But, its even accepting b=a then a=7 .

Refer below for the problem statement:

Challenge [Declare JavaScript Variables](https://www.freecodecamp.com/challenges/declare-javascript-variables#?solution=%2F%2F%20Example%0Avar%20ourName%3B%0Avar%20myName%3B%0A%2F%2F%20Define%20myName%20below%20this%20line%0A%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
// Example
var ourName;
var myName;
// Define myName below this line


```

### PR DESCRIPTION
**Pull request by erictleung**
Tested locally. Passes `npm run test-challenges`. Closes #7806. Crude fix but it works.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -100,6 +100,14 @@
         "Assign the contents of <code>a</code> to variable <code>b</code>."
       ],
       "releasedOn": "January 1, 2016",
+      "head": [
+        "if (a != 'undefined') {",
```
Comment (BerkeleyTrue) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: I think you meant `(typeof a != 'undefined')`?

**Comment by alistermada** [comment]
Would this also fix #7890? The code in that issue passes even though `a` is undefined. This happens when `a` is previously assigned the value of 7 and that value is stored in memory when tests are run.

**Comment by erictleung** [comment]
@alistermada I couldn't replicate even problem on my local instance so I figured I'd test it on the live site whenever my changes get pushed.

**Comment by BerkeleyTrue** [comment]
@erictleung So many PR's!! Thanks!

**Pull request merged**